Configuration/Geometry/README.md

```diff
@@ -12,6 +12,7 @@ Calorimeters:
 * C4: HGCal (v9) + Phase2 HCAL and EB
 * C6: HGCal (v9) + HFNose + Phase2 HCAL and EB
 * C8: HGCal (v10 post TDR HGCal Geometry) + Phase2 HCAL and EB + Tracker cables in calorimeter region
+* C9: HGCal (v11 post TDR HGCal Geometry same as V8 + corner centering for HE part)
```

Comment:
did it really mean to say "same as V8" (C3 defined in 2016), or was it supposed to say "same as C8"?

Reply:
I think it should be C8. @bsunanda ?

Reply:
Yes it should be C8 - I shall correct this when I make the new PR with updated HGCal with HFNose